docs/home/LIFECYCLE.md

```diff
@@ -0,0 +1,14 @@
+# Epiphany platform lifecycle
```

Comment:
Maybe also add a note that we current support the quarterly minor releases for a year with new patch versions but that the patch versions itself fall under the minor releases.